charts/kong-collectorapi/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
```

Comment:
It is usually recommended to not create Pod instances like this. A `Job` is more suitable and manageable for such use-cases.